vmc/resource_vmc_sddc.go

```diff
@@ -607,6 +606,9 @@ func resourceSddcUpdate(d *schema.ResourceData, m interface{}) error {
 			return resource.NonRetryableError(resourceSddcRead(d, m))
 		})
 	}
+	if d.HasChange("size") {
```

Comment:
did you want to update the 'size' param name too?

Reply:
I don't think we should update it. The name should match the one from the spec since we don't want to have inconsistency in what it is called in Eve and what the TF provider documentation calls it. I can follow up with the VMC team on that. Till then we should call it 'size'.